### PR TITLE
Locate poi

### PR DESCRIPTION
**Pull request by Dezedene**
Informations in point of interest window, and transition.

### CODE REVIEW

File: src/Components/PoiInformations.js [hunk 1]
```diff
@@ -4,49 +4,60 @@ import { connect } from 'react-redux';
...
+      role="button"
+      tabIndex="0"
+    >
+      <h1>{specificPoiInfos.name}</h1>
+      <hr />
+      <p>Adresse</p>
+      <p>code postal ville</p>
+      <p>Distance</p>
+      <div>
+        <img src={specificPoiInfos.picture_url} className={InformationPoiInfos ? 'informationPicture' : 'informationPicture-Bottom'} alt="table ping pong" />
```
Comment (erabin05) [inline src/Components/PoiInformations.js]: ```alt``` is not a dynamic value, I don't think you will always have a picture of a ping pong table

**Review by erabin05 — CHANGES_REQUESTED** [review]
- the ```alt``` must be dynamic

File: src/Components/PoiInformations.js (hunk 1 above)
Comment (Dezedene) [reply src/Components/PoiInformations.js]: Modification ok

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**